src/ucs/stats/client_server.c

```diff
@@ -299,7 +300,7 @@ ucs_stats_server_entity_get(ucs_stats_server_h server, struct sockaddr_in *addr)
 
 static void ucs_stats_server_entity_put(stats_entity_t * entity)
 {
-    if (__sync_fetch_and_add(&entity->refcount, -1) == 1) {
+    if (__sync_fetch_and_sub(&entity->refcount, 1) == 1) {
```

Comment:
why just not `ucs_atomic_fadd32(&entity->refcount, 1)`?

Reply:
original code was created about 5 years ago, ucs_atomic_XXX were created later

I will replace all sync_fetch calls by ucs_atomic in additional PR (there are more such calls)

Reply:
got it, thanks!